www/include/monitoring/downtime/AddDowntime.php

```diff
@@ -283,8 +283,30 @@
         )
     );
 
+    $durationScale = array(
```

Comment:
This duration is hardcoded and does not correspond to the one chosen by the user
It may need to be chosen by the user, but the current value does not fit this one

Reply:
https://github.com/centreon/centreon/pull/7606#discussion_r351911993

Reply:
It seems, that you're too fast for me :)